integration/shareddata/composites.go

```diff
@@ -66,18 +66,32 @@ var DocumentsDoubles = &Values[string]{
 	handlers: []string{"pg", "tigris"},
 	validators: map[string]map[string]any{
 		"tigris": {
-			"$tigrisSchemaString": tigrisSchema(`"type": "object", "properties": {"v": {"type": "number"}}`),
+			"$tigrisSchemaString": `{
+				"title": "%%collection%%",
+				"primary_key": ["_id"],
+				"properties": {
+					"v": {
+						"type": "object", 
+						"properties": {
+							"foo": {"type": "object", "properties": {"bar": {"type": "string"}} }, 
+							"v": {"type": "number"}
+						}
+					},
+					"_id": {"type": "string"}
+				}
+			}`,
 		},
 	},
 	data: map[string]any{
-		"document-double":          bson.D{{"v", 42.13}},
-		"document-double-whole":    bson.D{{"v", 42.0}},
-		"document-double-zero":     bson.D{{"v", 0.0}},
-		"document-double-max":      bson.D{{"v", math.MaxFloat64}},
-		"document-double-smallest": bson.D{{"v", math.SmallestNonzeroFloat64}},
-		"document-double-big":      bson.D{{"v", doubleBig}},
-		"document-double-empty":    bson.D{},
-		"document-double-null":     nil,
+		"document-double":           bson.D{{"v", 42.13}},
+		"document-double-whole":     bson.D{{"v", 42.0}},
+		"document-double-zero":      bson.D{{"v", 0.0}},
+		"document-double-v-not-set": bson.D{{"foo", nil}},
```

Comment:
How that's different from `document-double-empty`? And we sneak it into doubles?